template/src/ontology/catalog-v001.xml.jinja2

```diff
@@ -2,10 +2,15 @@
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
 <group id="Folder Repository, directory=, recursive=false, Auto-Update=false, version=2" prefer="public" xml:base="">
 {% if project.import_group is defined %}
+  {% if project.import_group.use_base_merging %}
+  <uri name="{{project.uribase}}/{% if project.uribase_suffix is not none %}{{ project.uribase_suffix }}{% else %}{{ project.id }}{% endif %}/imports/merged_import.owl" uri="imports/merged_import.owl"/>
+  <uri name="{{project.uribase}}/{% if project.uribase_suffix is not none %}{{ project.uribase_suffix }}{% else %}{{ project.id }}{% endif %}/imports/merged_import.obo" uri="imports/merged_import.obo"/>
```

Comment:
obo omports not needed, nowhere. I think this can be scrapped ones and for all also from the `else` part.

Reply:
Yes, I noticed that. I decided to keep the OBO entry for consistency with the individual imports, but happy to go the other way and instead remove all OBO catalog entries for OBO imports.